src/entity2/user.ts

```diff
@@ -0,0 +1,49 @@
+import { Entity, PrimaryGeneratedColumn, Column, BaseEntity } from 'typeorm';
+
+@Entity()
+export class User extends BaseEntity {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column({ unique: true })
+    username: string;
+
+    @Column({ unique: true })
+    email: string;
+
+    @Column({ nullable: true, unique: true })
+    oidc_subject: string;
```

Comment:
If this is NULL then I assume the object does not represent any kind of OIDC principal.
In that case do we need an alternative way to authenticate into the account such as a password? Or are such accounts only for system stuff such as batch jobs?

Reply:
There will be system users I suspect... Right now there's a test user and I am allowing for the possibility to roll a local authentication strategy which is looking increasing like it might be needed.